cli_reference/helm-cli/getting-started-with-helm-on-openshift-container-platform.adoc

```diff
@@ -0,0 +1,16 @@
+[id="getting-started-with-helm-on-openshift"]
+
+= Getting started with Helm on Openshift
```

Comment:
s/Openshift/OpenShift/ - although I don't think it's necessary to use "on OpenShift" as it should be implied. Same goes for the _topic_map.yml entry. I suggest, "Getting started with Helm." If you do include it, I would use the {product-title} variable (as you do later in "Creating a custom Helm chart."